src/ape/contracts/base.py

```diff
@@ -674,7 +675,11 @@ def query(
         )
         columns_ls = validate_and_expand_columns(columns, ContractLog)
         data = map(partial(extract_fields, columns=columns_ls), contract_events)
-        return pd.DataFrame(columns=columns_ls, data=data)
+        df = pd.DataFrame(columns=columns_ls, data=data)
+        if "event_arguments" in columns_ls:
+            event_arguments = df["event_arguments"].apply(pd.Series)
+            df = pd.concat([df.drop("event_arguments", axis=1), event_arguments], axis=1)
```

Comment:
Add a comment about what this does please

Reply:
left a note above this block